### DISCUSSION TITLE
No working version for Linux distro Pop!_OS

### DISCUSSION THREAD

**Discussion by RelicCornhusk** [moneymanagerex/moneymanagerex#3716] [Q&A]
Currently using Pop!_OS 21.04 and I could not find any version that would work with my OS. If anyone had success installing it, please let me know, as MMEX seems the best solution for my finances at the moment.

**Comment by Timp58** [comment]
I have also been unable to get a version to work on Pop OS.  The closest I came was using Wine with a 64 bit portable version but the screen windows so tiny to be unusable, I don't know if this is a wine problem or Pop OS.
The versions for Ubuntu are so old they are of very little use.  I have even considered building the app myself but the instructions for doing so merely say to do the same as for the macOS version but there are clearly Mac-specific lines in the commands.  It seems this documentation is quite out of date.
I use MMEX all the time on Windows but I am trying to make more use of Linux and not being able to use MMEX is a big drawback

**Reply by EBiForE** [reply]
Same problem, I abandoned Money Manage Ex, they removed support for both Android and Linux which are the only operating systems I use

**Reply by Timp58** [reply]
And yet the web site continues to claim the program is cross platform.

**Reply by vomikan** [reply]
That's the way it is. The program works on many Linux distributions. But at the moment we are not distributing binary packages, as it was before.

**Comment by andrepg** [comment]
They had a problem a few months ago with TravisCI becoming a paid service for OpenSource Projects with many hours of build.

For now it seems that only way to get installed in Linux is building from sources. You could use the Build instructions and [this comment](https://github.com/moneymanagerex/moneymanagerex/issues/3853#issuecomment-939563769)
